src/App.js

```diff
@@ -15,6 +19,8 @@ function App() {
           Learn
         </a> */}
       </header>
+      <Body />
```

Comment:
Why did you add a body? I dont know how this will be used

Reply:
Body was going to be used as a place holder to create a gap between header and footer, but a different method was used.